tests/js/sentry-test/select-new.js

```diff
@@ -34,16 +34,12 @@ export function findOption(wrapper, {value, label} = {}, options) {
 
 export function selectByLabel(wrapper, label, options = {}) {
   openMenu(wrapper, options);
-  findOption(wrapper, {label}, options)
-    .at(options.at || 0)
-    .simulate('click');
+  findOption(wrapper, {label}, options).at(0).simulate('click');
 }
 
 export function selectByValue(wrapper, value, options = {}) {
   openMenu(wrapper, options);
-  findOption(wrapper, {value}, options)
-    .at(options.at || 0)
-    .simulate('click');
+  findOption(wrapper, {value}, options).at(0).simulate('click');
```

Comment:
These need to be `0` as select inputs will likely not have duplicate options and using `options.at` here made operating on select boxes that were not first in the page impossible.